### PR TITLE
Subscribed cat stream

### PR DESCRIPTION
**Pull request by Justin-Holmes**
https://www.pivotaltracker.com/story/show/154940601

### CODE REVIEW

File: apps/ello_v3/lib/ello_v3/resolvers/pagination_helpers.ex [hunk 1]
```diff
@@ -9,7 +9,7 @@ defmodule Ello.V3.Resolvers.PaginationHelpers do
   def is_last_page(args, structs, filer_slop \\ @filter_slop)
   def is_last_page(_, [], _), do: true
   def is_last_page(%{per_page: requested}, structs, filter_slop) do
-    if requested - filter_slop > length(structs), do: true, else: false
+    if requested - filter_slop >= length(structs), do: true, else: false
```
Comment (Justin-Holmes) [inline apps/ello_v3/lib/ello_v3/resolvers/pagination_helpers.ex]: We may need to rethink how we do this. I changed it to fix my specs.

Comment (alanpeabody) [reply apps/ello_v3/lib/ello_v3/resolvers/pagination_helpers.ex]: Why did it need to change? If we need to change the slop by one passing in the third variable might be better.

There are definitly cases where we don't know for sure if it is the last page and return false, even though a subsequent request will return nothing. Is that the case in the specs?

Comment (Justin-Holmes) [reply apps/ello_v3/lib/ello_v3/resolvers/pagination_helpers.ex]: Yep, the way I set my tests up the `isLastPage` variable was wrong. I'm not sure how to pass in the `filter_slop` variable without adding unnecessary function arguments to our resolvers and stuff. I may just tweak the test to accommodate our predetermined `filter_slop`.

File: apps/ello_v3/lib/ello_v3/resolvers/category_post_stream.ex [hunk 2]
```diff
@@ -24,7 +25,7 @@ defmodule Ello.V3.Resolvers.CategoryPostStream do
         {:ok, %{
           id:    category.id,
           slug:  category.slug,
-          posts: search.results,
+          posts: track(search.results, args, kind: "#{String.downcase(category.name)}_trending"),
```
Comment (alanpeabody) [inline apps/ello_v3/lib/ello_v3/resolvers/category_post_stream.ex]: How do we track category post views now? I think this should maybe be category_trending with an additional `id: category.id` property.

Comment (Justin-Holmes) [reply apps/ello_v3/lib/ello_v3/resolvers/category_post_stream.ex]: @alanpeabody Yeah, I can do that. I was envisioning an easier approach where the data would be flattened but probably better to keep it consistent.

**Pull request merged**